### PR TITLE
[TP-11413] - Travel insurance registration confirmation email copy

### PR DESCRIPTION
**Pull request by ryanharkins**
Screenshot 2020-05-11 at 16 48 07

### CODE REVIEW

**Comment by jordanlg27** [comment]
https://maps.tpondemand.com/entity/11413

**Comment by masbuild** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">There is no Targetprocess ticket associated to this PR. Please include the link to the corresponding task (TP-XXXX). If no ticket matches these changes consider creating one and linking it here.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">There are code changes, but no corresponding tests. Please include tests if this PR introduces any modifications in behavior. If this is a refactor PR, please ignore the message.Refer to our mas standards guides about opening a PR<a href="link-to-the-mas-standards-about-opening-a-pr">here</a></td>
    </tr>
  </tbody>
</table>

At Money Advice Service, we use Danger gem, to ensure that our commits and pull requests follow [standards](https://github.com/moneyadviceservice/mas-standards/blob/master/Dangerfile). If you think one of the points here has been raised in error (or shouldn't be in our standards), open an issue or pull request on [our Danger config](https://github.com/moneyadviceservice/mas-standards).

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: app/assets/javascripts/modules/WizardForm.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ define(['jquery'], function($) {
   function disableOrEnableSubmitButton(step){
     $submitButton.attr('disabled', step.find('input:checked').length <= 0)
     if($currentStepNumber > 0){
-      $prevButton.show()
+      $prevButton.css('display', 'inline-block')
```
[the remark this answers is not in the document]
Comment (ryanharkins) [reply app/assets/javascripts/modules/WizardForm.js:51]: Yes, `.show()` adds a display block to the element and it pushes down anything that's next to the button. I'll update the hide call to use the css call then

[the remark this answers is not in the document]
Comment (kiggundu) [reply app/assets/javascripts/modules/WizardForm.js:51]: cool

**Pull request merged**